java/vector/src/main/java/org/apache/arrow/vector/util/DecimalUtility.java

```diff
@@ -136,8 +133,9 @@ public static StringBuilder toStringWithZeroes(long number, int desiredLength) {
     return str;
   }
 
-  public static BigDecimal getBigDecimalFromArrowBuf(ArrowBuf bytebuf, int startIndex, int scale) {
+  public static BigDecimal getBigDecimalFromArrowBuf(ArrowBuf bytebuf, int index, int scale) {
```

Comment:
Thanks for fixing the consistency. Can you add the doc explaining `index` is not the byte index, but the value index?

Reply:
Yeah, adding some docs would be nice here